app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -16,4 +17,68 @@ class BloodSugarEntrySheet : BottomSheetActivity() {
 
     setContentView(R.layout.sheet_blood_sugar_entry)
   }
+
+  override fun setBloodSugarSavedResultAndFinish() {
+    TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Not important to change this commit, but could you please update the template in your IDE to remove the following section?

```
//To change body of created functions use File | Settings | File Templates.
```

Reply:
Done